bootstrap/overlay/etc/systemd/system/multi-user.target.wants/vmtoolsd.service

```diff
@@ -0,0 +1 @@
+../vmtoolsd.service
```

Comment:
@vrutkovs does this service already run only conditionally on vmsphere?

Reply:
I don't think so - the service is provided from RPM and effectively a noop on other platforms. Also it doesn't fail so its safe to have it running everywhere.

I guess we could add a dropin to run only on vsphere later